static/app/views/performance/browser/webVitals/utils/queries/storedScoreQueries/useTransactionSamplesWebVitalsScoresQuery.tsx

```diff
@@ -50,6 +50,14 @@ export const useTransactionSamplesWebVitalsScoresQuery = ({
     sortableFields: filteredSortableFields as unknown as string[],
   });
 
+  const mutableSearch = new MutableSearch([
+    'transaction.op:pageload',
+    'has:measurements.score.total',
+  ]).addStringFilter(`transaction:"${transaction}"`);
```

Comment:
Why can't we just slap this into the original array argument to MutableSearch?

Reply:
When we add the transaction filter inside the `MutableSearch` array arg, the `transaction` filter value doesn't get escaped. Using `addStringFilter` escapes the value by default. We need to escape the value because sometimes the transaction name has an `*` character which becomes a wildcard search if not escaped.